modules/combined/examples/geochem-porous_flow/forge/porous_flow.i

```diff
@@ -630,6 +475,11 @@
     density = 2750.0
     specific_heat_capacity = 900.0
   [../]
+  [./relperm] # this is necessary because of the use_mobility = true used in the PorousFlowPeacemanBorehole objects
+    type = PorousFlowRelativePermeabilityConst
+    at_nodes = true
```

Comment:
Is the `at_nodes` required? If it is, remind me to fix this so that it isn't

Reply:
Yea, `at_nodes` is needed, otherwise a 

> ```
> [blah]
>   blah
>   blah
> []
> ```
> 
> 
> while others have
> 
> ```
> [./blah]
>   blah
>   blah
> [../]
> ```

I just submitted a new issue on this (#17024) - i think it's more widespread than the test/example files in this PR

Reply:
?  The above comment about #17024 was nothing to do with the `at_nodes` remark, sorry.

The `at_nodes` param is needed, otherwise both the Action and the AddJoiner will create the `at_nodes = False` version.   Hope that makes sense!

Reply:
I just submitted an issue on this `at_nodes` thing - #17025